docroot/themes/custom/twentyeighteen/sass/components/call-to-action/call-to-action.twig

```diff
@@ -0,0 +1,9 @@
+<div class= "call-to-action-block">
+  <div class= "call-to-action-img">
+  <img src="https://static.pexels.com/photos/59919/pexels-photo-59919.jpeg">
+  </div>
+  <div class= "call-to-action-text">
+  <h1>Connect</h1>
+  <p>Interested in sponsoring or exhibiting?</p>
+  </div>
+</div>
```

Comment:
Suggested markup: 
`<div class= "call-to-action">`
    `<div class= "call-to-action__img">`
      `<img src="https://static.pexels.com/photos/59919/pexels-photo-59919.jpeg">`
    `</div>`
    `<div class= "call-to-action__text">`
    `<h2 class="call-to-action__text-header">Connect</h1>`
    `<p class="call-to-action__text-details">Interested in sponsoring or exhibiting?</p>`
 `</div>`
`</div>`

Reply:
arg, it didn't take my indentation into account.